src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -67,8 +89,70 @@ def execute_mdtest(self):
         # Create a pool
         self.pool.create()
 
-        # Run Mdtest
+    def create_cont(self):
+        """Create a TestContainer object to be used to create container."""
+        # TO-DO: Enable container using TestContainer object,
+        # once DAOS-3355 is resolved.
+        # Get Container params
+        #self.container = TestContainer(self.pool)
+        #self.container.get_params(self)
+
+        # create container
+        # self.container.create()
+        env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
```

Comment:
I know this is also a copy of other code but it seems a odd way of doing things.  This line will create a Dfuse() object, read the device setting from the yaml file and close the object again, there's no need to create a Dfuse object to do this, can't it be done by a call to DaosContainer().create() instead?

Even if do you want to have this coded this way why not call .get_default_env() in self.dfuse?

Reply:
Yep, I am aware it is not one of the cleanest ways of implementing it. But this bit is just temporary. It should go away once the TO-DO bit mentioned above is implemented.
And regarding the second question, this is initiated here bcoz start_dfuse() needs it's own set of env which is slightly different that what container create need here. If I do it inside start_dfuse, it could interfere with those env and cause issues.

Reply:
I can't see a TO-DO above, can you point it out to me please?

get_default_env() just returns a dict, it doesn't modify anything so I don't see why you need a whole new object rather than the current one to call it.  This code isn't even calling dfuse is it?

Reply:
Sorry, I was wrong about the return type.

You haven't explained why this line can't self 'env = self.get_default_env()'